### DISCUSSION TITLE
Analog mapping to face buttons

### DISCUSSION THREAD

**Discussion by Edgylatterdaysaint** [rafaelvcaetano/melonDS-android#1300] [Ideas]
TL;DR: Having the ability to map the analog stick to the face buttons for FPS games (Like MPH) that have an aim option for them, and having multiple things assigned to one button if desired, as well as custom input mappings for specific games.

Hi! I recently got. Razer Kishi for my Galaxy Note 10 Plus, and was starting to get back into my games and such with emulation. I found Melon DS, and it's way better than Drastic, which is awesome! I've loved it thus far.

I also recently got Star Wars Battlefront 2 on PSP. One of the control options is using the face buttons to aim. I decided to make a custom control mapping in Ppsspp for the analog stick on my Kishi to be mapped to my face buttons at the same time, this way it's like I'm aiming analog. It works perfectly. 

I found that Metroid Prime Hunters has a similar control option. But to my dismay, I cannot seem to 
1, map my stick to my face buttons, and 
2, have multiple assignments to the same button, like having L2 be L and L1 be L at the same time, or, the right stick and face buttons both being mapped as A B X Y, and 
3, having custom control mappings for specific games.

There are a lot of solid shooters on DS, and I'm sure most have this control option. I think having the feature to map the control stick to face buttons, as well as multiple assignments to a single button would be great for Melon DS! 

Thanks for reading all of that!
